app/models/friendship.rb

```diff
@@ -1,6 +1,6 @@
 class Friendship < ApplicationRecord
   belongs_to :follower, class_name: 'User'
   belongs_to :followee, class_name: 'User'
-  has_many :party_participants
+  has_many :party_participants, dependent: :destroy
```

Comment:
This was the question I had for you!  I'm glad you were able to figure out which kind of dependency it required!

Reply:
Yes! I am still researching "dependency" and "inverse of", but for this one, if I delete someone as a friend, it should also delete them from the party participants, so that makes sense as a dependent destroy => Thought this might update on my database branch updates as I deal with the user friending and the party getting participants